.gitignore

```diff
@@ -12,3 +12,4 @@ Package.resolved
 
 # DocC generated files
 .docc-build
+DerivedData
```

Comment:
Would you move it up to the Xcode section and delete one of the `.xcode`s?

Reply:
I moved it under .xcode